operator/api/v1alpha1/keptnappversion_types.go

```diff
@@ -231,10 +231,14 @@ func (v KeptnAppVersion) GetVersion() string {
 	return v.Spec.Version
 }
 
-func (v KeptnAppVersion) GetSpanName(phase string) string {
+func (v KeptnAppVersion) GetSpanKey(phase string) string {
 	return fmt.Sprintf("%s.%s.%s.%s", v.Spec.TraceId, v.Spec.AppName, v.Spec.Version, phase)
 }
 
+func (v KeptnAppVersion) GetSpanName(phase string) string {
+	return phase
```

Comment:
```suggestion
	return fmt.Sprintf("%s/%s", v.Spec.AppName, phase)
```

Reply:
In the App Span we don't use the App name